pkg/cmd/validator/validator.go

```diff
@@ -1020,6 +1060,15 @@ func applyPlugins(c *cfg.Config, vc *components.ValidatorConfig) error {
 		}
 	}
 
+	if vc.MaasPlugin.Enabled {
```

Comment:
Super nit... but can you please alphabetize the plugin code blocks here. They're alphabetized in most other places already.